src/webrtc.ts

```diff
@@ -385,7 +386,7 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
   }
 
   prepareSession (options: SessionOpts): StanzaMediaSession | undefined {
-    if (this.sdpOverXmpp) {
+    if (options.sid && this.sessionsMap[options.sid]) {
```

Comment:
Originally we had a check here for `this.sdpOverXmpp` but the issue that was found on the corresponding ticket is that value has the potential to change when it should not.  Softphone call comes in and is using SDP over XMPP, `this.sdpOverXmpp` is set to true. Screen recording comes in and is _not_ using SDP over XMPP, `this.sdpOverXmpp` is set to false and coming to this point, results in this function not getting skipped.